src/client/src/containers/RightSidebar/index.tsx

```diff
@@ -196,7 +212,11 @@ class RightSidebar extends React.Component<Props, IRightSidebarState> {
                   optionsData={backendOptions}
                 />
                 <div className={styles.sortablePages}>
-                  {showPages && <SortablePageList />}
+                  {showPages && (
+                    <SortablePageList
+                      handleResetPages={this.resetAllPages.bind(this)}
```

Comment:
If you transform the method `resetAllPages` into an arrow function you won't need to bind this explicitly 😁

Reply:
LOL thanks mate